src/notebooks/filesystem-tutorial.ipynb

```diff
@@ -1728,9 +1863,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.8-final"
+   "version": "3.8.8"
```

Comment:
3.8 instead of 3.9?

Reply:
🤣 I need to update my local environment, but yeah, it is interesting that it still says 3.8 even before my changes.